components/Dialogs/README.md

```diff
@@ -223,11 +248,89 @@ MDCAlertControllerThemer.applyScheme(scheme, to: alertController)
 // Step 1: Import the DialogThemer extension
 #import "MaterialDialogs+DialogThemer.h"
 
-// Step 2: Create or get a alert scheme
-MDCAlertScheme *alertScheme = [[MDCAlertScheme alloc] init];
+// Step 2: Create or get a Material container scheme
+MDCContainerScheme *scheme = [[MDCContainerScheme alloc] init];
+
+// Step 3 (optional): Customize the default theme by providing custom color or typography schemes:
+scheme.colorScheme = myColorScheme
+scheme.typographyScheme = myTypographyScheme
+
+// Step 4: Use the Material alert themer to theme an MDCAlertController instance
+[alertController applyThemeWithScheme:scheme];
+
+// Step 4: Alternatively, Use Material dialog presentation themer to theme your UIViewController instance:
+[myDialogViewController.mdc_dialogPresentationController applyThemeWithScheme: scheme];
+
+```
+<!--</div>-->
+
+<!-- Extracted from docs/action-theming.md -->
+
+### Action Theming
+
+Actions in MDCAlertController have emphasis which afects their theming.
+High emphasis actions generate contained buttons, medium emphasis actions generate outlined buttons and low emphasis actions generate text buttons.
+
+<div class="article__asset article__asset--screenshot">
+  <img src="docs/assets/dialogButtons.png" alt="Dialogs Actions" width="320">
```

Comment:
Accessibility: Please describe the contents of the image. As a good guideline, I should be able to read the `alt` text without ever seeing the image and have a good understanding of what is being presented.  Examples:

* "An alert presented with a title, body, high-emphasis 'OK' button and low-emphasis 'Cancel' button."
* "A 'device offline' alert showing a medium-emphasis 'OK' button."